.circleci/config.yml

```diff
@@ -219,3 +219,23 @@ workflows:
                 command: |
                          curl -X POST -H "Content-type:application/json" --data "{'text':':error: $CIRCLE_PROJECT_REPONAME/$CIRCLE_BRANCH - $CIRCLE_PULL_REQUEST validation failed on *Node:${CIRCLE_JOB/job_pr-validator-node-/''}*. View <$CIRCLE_BUILD_URL|build>. cc: <@$CIRCLE_USERNAME>', 'mrkdwn':true}" $SLACK_WEBHOOK
                 when: on_fail
+
+  workflow_nightly-e2e:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
```

Comment:
can we run the cron at like 9am PST? So if it fails, it's the beginning of our workday and we can address it